<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grafiek</class>
 <widget class="QDialog" name="grafiek">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>817</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>821</width>
     <height>721</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QWidget" name="verticalWidgetTemp" native="true">
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="verticalWidgetHum" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="verticalWidgetPres" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Terug</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
